src/openrct2-ui/input/KeyboardShortcuts.cpp

```diff
@@ -159,22 +159,22 @@ void KeyboardShortcuts::Set(int32_t key)
 
 int32_t KeyboardShortcuts::GetFromKey(int32_t key)
```

Comment:
This should return the new type instead of int32_t. that will reduce the amount of static casts.